docs/source/asr/configs.rst

```diff
@@ -256,3 +291,104 @@ For example, a decoder config corresponding to the encoder above would look like
       feat_in: *enc_filters
       vocabulary: *labels
       num_classes: 28   # Length of the vocabulary list
+
+Citrinet
+~~~~~~~~
+
+The `Citrinet <./models.html#Citrinet>`__ and `QuartzNet <./models.html#QuartzNet>`__ models are very similar, and as such the components in their configs are very similar as well. Citrinet utilizes Squeeze and Excitation, as well as sub-word tokenization, in contrast to QuartzNet.
+
+Both architectures use the ``ConvASREncoder`` for the ``encoder``, with parameters detailed in the table above.
+The encoder parameters include details about the Citrinet-C encoder architecture, including how many
+filters are used per channel (C). The Citrinet-C configuration is a shortform notation for Citrinet-21x5xC, such that B = 21 and R = 5 are the default and should generally not be changed.
+
+To use Citrinet instead of QuartzNet, please refer to the ``citrinet_512.yaml`` configuration found inside the ``examples/asr/conf/citinet`` directory. While the configs for Citrinet and QuartzNet are similar, we note the additional flags used for Citrinet below
+
++-------------------------+------------------+---------------------------------------------------------------------------------------------------------------+---------------------------------+
```

Comment:
How about giving a reference to the argument docs generated from doc strings for this model?
We can move any descriptions there to avoid duplication. It makes it easier to update and maintain them.